### PR TITLE
Unifying transform/translate3d description and agruments list with toggleClass/toggleAttribute

### PR DESCRIPTION
**Pull request by nazar-pc**

### CODE REVIEW

File: 1.0/docs/devguide/utility-functions.md [hunk 1]
```diff
@@ -54,13 +54,13 @@ provides a set of useful convenience functions for instances to use.
 
 *   `isDebouncerActive(jobName)`. Returns true if the named debounce task is waiting to run.
 
-*   `transform(transform, node)`. Applies a CSS transform to the specified node,
-    or this element if no node is specified.
+*   `transform(transform, [node])`. Applies a CSS transform to the specified node,
```
Comment (ebidel) [inline 1.0/docs/devguide/utility-functions.md]: It's Google's JS doc style to use `opt_Node`. `[node]` can be confused for an array argument.

@arthurevans we should probably rely on the API docs for these methods: http://polymer.github.io/polymer/

Comment (ebidel) [inline 1.0/docs/devguide/utility-functions.md]: nvm on the `[]`. I see we're already using it on this page and we should be consistent with it.

Comment (arthurevans) [inline 1.0/docs/devguide/utility-functions.md]: Yep, we need deep-linking for the API docs before we can really point people to them. See:

https://github.com/PolymerElements/iron-component-page/issues/28

Once we get that I'd like to link to the API docs more aggressively.

**Comment by arthurevans** [comment]
LGTM.

**Pull request merged**